### PR TITLE
Added a Blog Button

### PR DESCRIPTION
**Pull request by BAAHUBALI7781**

### CODE REVIEW

File: src/views/AddBlog.vue [hunk 1]
```diff
@@ -1,9 +1,126 @@
...
-    <h1 class="Display-4 text-center">Add here</h1>
-</template>>
+    <div style="margin:0 auto">
+    <nav class="main_nav">
+        <ul style="display: inline-block;">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Edit</a></li>
+            <li><a href="#">Format</a></li>
+            <li><a href="#">Help</a></li>
+		<img class="icon" src="../../src/assets/pilogo.png" width="39" />
```
Comment (shivam-51) [inline src/views/AddBlog.vue]: update this

Comment (BAAHUBALI7781) [reply src/views/AddBlog.vue]: It is already updated!

Comment (shivam-51) [reply src/views/AddBlog.vue]: pilogo.png needs to change

File: src/views/AddBlog.vue [hunk 2]
```diff
@@ -1,9 +1,126 @@
 <template>
-    <h1 class="Display-4 text-center">Add here</h1>
-</template>>
+    <div style="margin:0 auto">
+    <nav class="main_nav">
+        <ul style="display: inline-block;">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Edit</a></li>
+            <li><a href="#">Format</a></li>
+            <li><a href="#">Help</a></li>
```
Comment (shivam-51) [inline src/views/AddBlog.vue:8]: Why is this required? we already have a Navbar

Please run your changes in an updated branch

**Review by shivam-51 — CHANGES_REQUESTED** [review]

**Comment by shivam-51** [comment]
@praneetha3108 Why did you merge this?
